### PR TITLE
Better overlay for MI 11 Lite 5G

### PR DESCRIPTION
**Pull request by danielalmeida1481**
All credits go to @AndroPlus-org

### CODE REVIEW

**Comment by phhusson** [comment]
CI result:

I'll allow those:
```
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_burnInProtectionMaxRadius
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_burnInProtectionMinHorizontalOffset
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_burnInProtectionMaxHorizontalOffset
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_burnInProtectionMinVerticalOffset
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_burnInProtectionMaxVerticalOffset
```

You need  to keep the overlay position that was there before your change in overlay.mk:
```
Fatal: overlay.mk: Keep entries sorted
```

Those settings don't exist in AOSP:
```
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_enableSmartPixels
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_showHideNotchSettings
```

This setting doesn't belong to hardware device choice, but to ROM:
```
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_camera_sound_forced
Fatal: ./Xiaomi/Mi11Lite5G/res/values/config.xml: defines a non-existing attribute config_ui_enableFadingMarquee
```

**Comment by danielalmeida1481** [comment]
I think it's all fixed now

File: Xiaomi/Mi11Lite5G-Settings/res/values/config.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <!-- Defines the location of the fingerprint sensor on the device
+         0 = back
+         1 = front
+         2 = left side
+         3 = right side
+    -->
+    <integer name="config_fingerprintSensorLocation">3</integer>
+
+    <!-- Whether to show min refresh rate in display settings -->
+    <bool name="config_show_refresh_rate_controls">true</bool>
```
Comment (phhusson) [inline Xiaomi/Mi11Lite5G-Settings/res/values/config.xml]: This doesn't exist in AOSP

File: Xiaomi/Mi11Lite5G-Settings/res/values/config.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    <!-- Custom pickup sensor for doze customization -->
+    <string name="pickup_sensor" translatable="false">xiaomi.sensor.pickup</string>
```
Comment (phhusson) [inline Xiaomi/Mi11Lite5G-Settings/res/values/config.xml]: This doesn't exist in AOSP

File: Xiaomi/Mi11Lite5G-Settings/res/values/config.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+*/
+-->
+<resources>
+    <!-- Defines the location of the fingerprint sensor on the device
+         0 = back
+         1 = front
+         2 = left side
+         3 = right side
+    -->
+    <integer name="config_fingerprintSensorLocation">3</integer>
```
Comment (phhusson) [inline Xiaomi/Mi11Lite5G-Settings/res/values/config.xml]: This doesn't exist in AOSP, however what does exist in AOSP and doesn't seem to be set in this overlay is config_is_powerbutton_fps

File: Xiaomi/Mi11Lite5G-SystemUI/res/values/config.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <!-- FPSInfoService FPS node file path -->
+    <string name="config_fpsInfoSysNode" translatable="false">/sys/class/drm/sde-crtc-0/measured_fps</string>
```
Comment (phhusson) [inline Xiaomi/Mi11Lite5G-SystemUI/res/values/config.xml]: This doesn't exist in AOSP

File: Xiaomi/Mi11Lite5G-SystemUI/res/values/config.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    <!-- The CPU temperature divider, if needed -->
+    <integer name="config_cpuTempDivider" translatable="false">1000</integer>
```
Comment (phhusson) [inline Xiaomi/Mi11Lite5G-SystemUI/res/values/config.xml]: This doesn't exist in AOSP

**Comment by danielalmeida1481** [comment]
Ok now it should be all fixed

**Pull request merged**